### PR TITLE
TST: implement test_first

### PR DESCRIPTION
**Pull request by jbrockmendel**
cc @MomIsBestFriend there are three things about these tests that I think would make for good (separate) follow-ups if you're interested:

1) They are just begging to be parametrized
2) We don't have a systematic way of naming/organizing tests that are for two specific methods rather than one specific method, which is the idea behind tests.(frame|series).methods.  (see also: head/tail, first_valid_index/last_valid_index)
3) These tests could/should also be shared/parametrized over Series vs DataFrame, but we dont have a systematic home or naming convention for these.

### CODE REVIEW

**Comment by MomIsBestFriend** [comment]
@jbrockmendel I would love to open a PR parameterising these tests.

Choosing a naming convention seems too difficult for me, tbh

File: pandas/tests/frame/methods/test_first.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+"""
+Note: includes tests for `last`
```
Comment (mroeschke) [inline pandas/tests/frame/methods/test_first.py]: Could we just rename the file `test_first_and_last.py`?

File: pandas/tests/series/methods/test_first.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+"""
```
Comment (mroeschke) [inline pandas/tests/series/methods/test_first.py]: Could we just rename the file `test_first_and_last.py`?

**Review by mroeschke — APPROVED** [review]

**Comment by simonjayhawkins** [comment]
Thanks @jbrockmendel I think naming the module test_first as you originally did is more consistent. Maybe we could have a test_last.py file which is just a comment that directs the contributor/reader to test_first.py

**Pull request merged**